src/main/java/ch/njol/skript/SkriptEventHandler.java

```diff
@@ -190,27 +185,25 @@ public static void logTriggerEnd(final Trigger t) {
 
 	public static void addTrigger(final Class<? extends Event>[] events, final Trigger trigger) {
 		for (final Class<? extends Event> e : events) {
-			List<Trigger> ts = triggers.get(e);
-			if (ts == null)
-				triggers.put(e, ts = new ArrayList<>());
+			List<Trigger> ts = triggers.computeIfAbsent(e, k -> new ArrayList<>());

```

Comment:
Lambdas cause object creation, and the object might become garbage if stack allocation does not get it. This is to be avoided in functions that are called (very) often.

Reply:
Why doesn't this use Guava's Multimap anyway ?

Reply:
Up-to-date Guava has been exposed to plugins only for a few years. And besides, this might be faster. Benchmark before changing this, I guess...

Reply:
iirc Multimap is a quite old Guava class, let me check that.

Reply:
What about that, should I revert that to how it was? `triggers.computeIfAbsent` occur in SkriptEventHandler and EvtMoveOn now